mkosi/__init__.py

```diff
@@ -310,31 +311,31 @@ def run_prepare_script(state: MkosiState, build: bool) -> None:
     if build and state.config.build_script is None:
         return
 
-    bwrap: list[PathString] = [
+    options: list[PathString] = [
         "--bind", state.config.prepare_script, "/work/prepare",
         "--chdir", "/work/src",
     ]
 
     for src, target in finalize_sources(state.config):
-        bwrap += ["--bind", src, Path("/") / target]
+        options += ["--bind", src, Path("/") / target]
 
     if build:
         with complete_step("Running prepare script in build overlay…"), mount_build_overlay(state):
-            run_workspace_command(
-                state.root,
-                ["/work/prepare", "build"],
-                network=True,
-                bwrap_params=bwrap,
+            bwrap(
+                ["chroot", "/work/prepare", "build"],
+                tools=state.config.tools_tree,
+                apivfs=state.root,
+                scripts=dict(chroot=chroot_cmd(state.root, options=options, network=True)),
                 env=dict(SRCDIR="/work/src") | state.environment,
             )
             shutil.rmtree(state.root / "work")
     else:
         with complete_step("Running prepare script…"):
-            run_workspace_command(
-                state.root,
-                ["/work/prepare", "final"],
-                network=True,
-                bwrap_params=bwrap,
+            bwrap(
+                ["chroot", "/work/prepare", "final"],
```

Comment:
A renaming to `/work/prepare` ended up in the commit "Rename extra argument of invoke_apt() to packages"